src/modules/logger/logged_topics.cpp

```diff
@@ -104,6 +104,7 @@ void LoggedTopics::add_default_topics()
 	add_optional_topic("px4io_status");
 	add_topic("radio_status");
 	add_topic("rover_ackermann_guidance_status", 100);
+	add_topic("rover_ackermann_status", 100);
```

Comment:
Can you make that (and the one above) `add_optional_topic`?

Reply:
I'm not 100% sure what the difference is, but if i make the guidance_status optional it does not get logged. I suppose it is because it doesn't start getting published until a mission is started.

Reply:
Hmm is it maybe not advertised? 
https://github.com/PX4/PX4-Autopilot/blob/09f066a73ae1eb9e1d39afc1c444a1f3264b0a17/src/modules/vtol_att_control/vtol_att_control_main.cpp#L80-L81

Reply:
Yes that fixed it: [8eb9996](https://github.com/PX4/PX4-Autopilot/pull/23310/commits/8eb999695eb840c09f075fe80e1a1c657d99f133)